### ISSUE TITLE
Filtering tasks type by `indexSwap` return an error

### ISSUE THREAD

**Issue by gmourier** [meilisearch/meilisearch#2969]
**Describe the bug**

When I want to filter the tasks by the task type `indexSwap` I get an error.

**To Reproduce**

## Query

`GET /tasks?type=indexSwap`

## Response

```json
{
    "message": "Query deserialize error: `indexSwap` is not a type. Available types are `documentAdditionOrUpdate`, `documentDeletion`, `settingsUpdate`, `indexCreation`, `indexDeletion`, `indexUpdate`, `indexSwap`, `taskCancelation`, `taskDeletion`, `dumpExport`, `snapshot`.",
    "code": "bad_request",
    "type": "invalid_request",
    "link": "https://docs.meilisearch.com/errors#bad_request"
}
```

**Expected behavior**

I'm able to retrieve all `indexSwap` tasks

**Meilisearch version:** `v0.30.0-new-index-scheduler.beta.1`

**Comment by curquiza** [comment]
Thanks for the report @gmourier

**Comment by curquiza** [comment]
Fixed by #2763
